src/server/json_family.cc

```diff
@@ -1376,6 +1428,27 @@ void JsonFamily::MSet(CmdArgList args, ConnectionContext* cntx) {
   cntx->SendOk();
 }
 
+// JSON.MERGE key path value
+// Based on https://datatracker.ietf.org/doc/html/rfc7386 spec
+void JsonFamily::Merge(CmdArgList args, ConnectionContext* cntx) {
+  string_view key = ArgS(args, 0);
+  string_view path = ArgS(args, 1);
+  string_view value = ArgS(args, 2);
+
+  if (path != "." && path != "$") {
```

Comment:
both . and $ means root?

Reply:
@adiholden yes, redis stack server behaves similarly with both "." and "$" literals.